internal/stage7.go

```diff
@@ -43,7 +46,7 @@ func testType2(stageHarness *test_case_harness.TestCaseHarness) error {
 		return err
 	}
 
-	availableExecutables := []string{"cat", "cp", "mkdir", "my_exe"}
+	availableExecutables := []string{"cat", "cp", "mkdir"}
```

Comment:
(a) The `TODO` comment isn't removed
(b) Why are we removing `my_exe` from the tests here? Isn't that still a case that needs to be tested?

Reply:
@rohitpaulk Ah, I assumed the second line to "change the test to not use exec.LookPath for my_exe at least" meant that. Will add it back.

Reply:
Adding more context: previously we were mutating `PATH` of the tester process, so we could use `LookPath` to figure out where `my_exe` is located. After this change, we're only changing `PATH` for the child process, so `LookPath` won't work anymore - we'll need to do something more custom for this one